server/src/main/java/io/crate/expression/symbol/WindowFunction.java

```diff
@@ -43,27 +43,39 @@
 public class WindowFunction extends Function {
 
     private final WindowDefinition windowDefinition;
+    private final Boolean ignoreNulls;
 
     public WindowFunction(StreamInput in) throws IOException {
         super(in);
         windowDefinition = new WindowDefinition(in);
+        ignoreNulls = false;
```

Comment:
This will need to be streamed properly - including a version check.

```suggestion
        if (in.getVersion().onOrAfter(Version.V_4_7_0)) {
            ignoreNulls = in.readOptionalBoolean(); 
        } else {
            ignoreNulls = null;
        }
```

And `writeTo` must write it to the stream as well